### PR TITLE
Locales: Update Portuguese Brazilian Translations pt_BR

### PR DESCRIPTION
**Pull request by altiereslima**

### CODE REVIEW

File: locales/pt_BR/pcsx2_Iconized.po [hunk 1]
```diff
@@ -2,20 +2,21 @@
...
+# Altieres Lima a Silva <altieres.lima@gmail.com> 2019-2020
 msgid ""
 msgstr ""
 "Project-Id-Version: PCSX2 1.7.0\n"
 "Report-Msgid-Bugs-To: https://github.com/PCSX2/pcsx2/issues\n"
 "POT-Creation-Date: 2020-02-10 17:29+0000\n"
-"PO-Revision-Date: 2019-08-20 12:27-0300\n"
-"Last-Translator: Rafael Fontenelle <rffontenelle@gmail.com>\n"
+"PO-Revision-Date: 2020-20-19 18:26-0300\n"
+"Last-Translator: Altieres Lima a Silva <altieres.lima@gmail.com>; Rafael Fontenelle <rffontenelle@gmail.com>
```
Comment (turtleli) [inline locales/pt_BR/pcsx2_Iconized.po]: Missing `\n"` at the end of this line.

File: locales/pt_BR/pcsx2_Iconized.po [hunk 2]
```diff
@@ -2,20 +2,21 @@
 # Copyright (C) 2019 PCSX2 Dev Team
 # This file is distributed under the same license as the PCSX2 package.
 # Rafael Fontenelle <rffontenelle@gmail.com>, 2011-2017.
+# Altieres Lima a Silva <altieres.lima@gmail.com> 2019-2020
 msgid ""
 msgstr ""
 "Project-Id-Version: PCSX2 1.7.0\n"
```
Comment (lightningterror) [inline locales/pt_BR/pcsx2_Iconized.po]: 1.6.0

File: locales/pt_BR/pcsx2_Main.po [hunk 3]
```diff
@@ -2,21 +2,22 @@
 # Copyright (C) 2019 PCSX2 Dev Team
 # This file is distributed under the same license as the PCSX2 package.
 # Rafael Fontenelle <rffontenelle@gmail.com>, 2011-2017.
+# Altieres Lima a Silva <altieres.lima@gmail.com> 2019-2020
 msgid ""
 msgstr ""
 "Project-Id-Version: PCSX2 1.7.0\n"
```
Comment (lightningterror) [inline locales/pt_BR/pcsx2_Main.po]: Here as well.

File: locales/pt_BR/pcsx2_Main.po [hunk 4]
```diff
@@ -2,21 +2,22 @@
...
+"Last-Translator: Altieres Lima a Silva <altieres.lima@gmail.com>; Rafael Fontenelle <rffontenelle@gmail.com>; Guilherme Dias "
 "<guilhermedias000@gmail.com>\n"
 "Language-Team: \n"
 "Language: pt_BR\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n > 1);\n"
-"X-Generator: Poedit 1.8.8\n"
+"X-Generator: Poedit Poedit 2.2.4\n"
```
Comment (lightningterror) [inline locales/pt_BR/pcsx2_Main.po]: Extra Poedit word not needed.

**Comment by RedDevilus** [comment]
https://my.pcloud.com/publink/show?code=XZ2zsskZvxShDWviapRh8RaXHCdLyfI4ExFV

If u need help :)

**Pull request merged**